docs/products/kafka/concepts/kafka-tiered-storage.rst

```diff
@@ -5,7 +5,7 @@ Tiered storage in Aiven for Apache Kafka® enables more effective data managemen
 
 .. important:: 
 
-    Aiven for Apache Kafka® tiered storage is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at sales@Aiven.io.
+    Aiven for Apache Kafka® tiered storage is an :doc:`Early availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at sales@Aiven.io.
```

Comment:
This is capitalised here but in lower case in the other two. Which is it?

Reply:
Good catch! I've updated the text to lower case.